spec/components/icon_component_spec.rb

```diff
@@ -12,9 +12,15 @@
   it 'renders icon svg' do
     rendered = render_inline IconComponent.new(icon: :print)
 
-    expect(rendered).to have_css(
-      ".usa-icon use[href^='#{vc_test_request.base_url}'][href$='.svg#print']",
-    )
+    icon = rendered.at_css('.icon.usa-icon')
+    id = icon.attr(:id)
+    inline_style = rendered.at_css('style').text.strip
+
+    expect(icon).to be_present
+    expect(inline_style).to match(%r{##{id}\s{.+?}}).
+      and include('-webkit-mask-image:').
+      and include('mask-image:').
+      and match(%r{url\([^)]+/print-\w+\.svg\)})
```

Comment:
personal style, since this is `and` as method and not a keyword, I'd add the parens
```suggestion
      and(include('-webkit-mask-image:')).
      and(include('mask-image:')).
      and(match(%r{url\([^)]+/print-\w+\.svg\)}))
```

Reply:
Good call, updated in 5bd9688.